lib/matplotlib/axes/_base.py

```diff
@@ -4215,7 +4215,7 @@ def set_navigate_mode(self, b):
         """
         Set the navigation toolbar button status.
 
-        .. warning ::
+        .. warning::
             this is not a user-API function.
```

Comment:
can you fix the capitalization here while you're at it?

Reply:
Sure, done.